### PR TITLE
Update practical-vbd.Rmd

### PR DESCRIPTION
**Pull request by kcharniga**
Corrected spelling, one equation, and changed discussion questions at the end.

### CODE REVIEW

File: content/post/practical-vbd.Rmd [hunk 1]
```diff
@@ -108,7 +108,7 @@ TIME     <-        # Number of years to run the simulation for
 
 $$\ \frac{dSh}{dt}  = \mu_h N_h - \frac {\beta_h b}{N_h} S_h  I_v - \mu_h  S_h $$
 $$\ \frac{dIh}{dt}  = \frac {\beta_h b}{N_h}S_h I_v - (\gamma_h + \mu_h) I_h $$
-$$\ \frac{dRh}{dt}  = \gamma_h + I_h  - \mu_v I_v$$
+$$\ \frac{dRh}{dt}  = \gamma_h I_h  - \mu_h I_h$$
```
Comment (zkamvar) [inline content/post/practical-vbd.Rmd]: @kcharniga, my math is not so good, but please confirm that you mean to change the addition to a multiplication here.

**Comment by kcharniga** [comment]
Yes, I intended to change that equation.

**Comment by zkamvar** [comment]
Everything works, so I'm going to push it and make the changes live. Thank you!

**Comment by kcharniga** [comment]
Great, thanks Zhian!

**Pull request merged**